### PR TITLE
Update content and spacing, add link to get more themes

### PR DESCRIPTION
**Pull request by janfaracik**
**Before**
image

**After**
image

**What's changed**
* Fixed spacing 
* Changed label 'Built-In Themes' to 'Themes'
* Added a link to get more themes 
* Removed the help file for the checkbox - the checkbox text seems clear enough as is 

Would be good to get some thoughts - thanks :)

### CODE REVIEW

File: src/main/resources/io/jenkins/plugins/thememanager/ThemeManagerPageDecorator/global.jelly [hunk 1]
```diff
@@ -1,6 +1,9 @@
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form" xmlns:st="jelly:stapler" xmlns:th="/lib/theme-manager">
-  <f:section title="${%Built-in Themes}">
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form" xmlns:l="/lib/layout" xmlns:th="/lib/theme-manager">
+  <f:section title="${%Themes}">
```
Comment (timja) [inline src/main/resources/io/jenkins/plugins/thememanager/ThemeManagerPageDecorator/global.jelly:3]: the reason this was called Built-in Themes was to stop the double themes label

image

You could probably fix that by making the radio label visually hidden or call it something like select a theme

Comment (janfaracik) [reply src/main/resources/io/jenkins/plugins/thememanager/ThemeManagerPageDecorator/global.jelly:3]: Done, changed it to 'Select theme' 👍

**Review by timja — APPROVED** [review]

**Comment by timja** [comment]
Not sure what's going on with the build failure here

**Comment by timja** [comment]
I can't reproduce it with:
```
mvn --batch-mode --show-version --errors --no-transfer-progress --update-snapshots -Dmaven.test.failure.ignore -Dspotbugs.failOnError=false -Dcheckstyle.failOnViolation=false -Dcheckstyle.failsOnError=false -Penable-jacoco help:evaluate -Dexpression=changelist  clean install
```

same java and maven version

**Comment by timja** [comment]
Looks like https://bugs.openjdk.org/browse/JDK-8297889 fixed in 11.0.18

**Pull request merged**